.github/workflows/pr.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+
+concurrency:
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Run tests
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: plugin-best-practices-plugin:check -s
```

Comment:
let's simplify this to just `check`. In the case we eventually add more modules (which may never happen), it'll run the task on all projects.

Reply:
Good point, just updated. Checks still not running on this PR anyway. I'm baffled 🤷‍♂️